components/Partners/Partner.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { s3 } from 'common/constants/urls';
+import styles from './Partner.css';
+
+const Partner = ({ partner }) => (
+  <div className={styles.Partner}>
+    <a href={partner.url} target="_blank" rel="noopener noreferrer">
+      <img className={styles.logo} src={`${s3}${partner.logo}`} alt={partner.name} />
+    </a>
+  </div>
+);
+
+Partner.prototype.propTypes = {
```

Comment:
If you change this to a vanilla function rather than an arrow func, you can define proptypes ABOVE the function definition. I'll look into adding lint rules that prevent arrow functions in the components folder.

Also, I don't believe `.prototype` is necessary.